### DISCUSSION TITLE
ESLint cannot recognize FontAwesomeIcon import

### DISCUSSION THREAD

**Discussion by AloisSeckar** [FortAwesome/Font-Awesome#19557] [Q&A]
Using ESLint with [Nuxt config](https://github.com/nuxt/eslint-config) I am getting violation of [import/named](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/named.md) rule from 
`import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'`

I am using the latest `@fortawesome/vue-fontawesome` package (atm it is `3.0.3`). The import I copied [from the docs](https://fontawesome.com/docs/web/use-with/vue/add-icons) and also it works correctly when applied. So I am confused.

The obvious workaround is just to ignore this particular package via

```
"settings": {
  "import/ignore": [
    "*vue-fontawesome*"
  ]
}
```

in ESLint settings, but this doesn't feel right to me. Also I would like at least to understand what is wrong with that?

**Comment by tagliala** [comment]
Hi, could you please open an issue on https://github.com/FortAwesome/vue-fontawesome ?

**Reply by AloisSeckar** [reply]
Sure I could. I just wasn't sure if it is an actuall issue or just something I need to do differently, so I wanted to ask first.
